lib/modules/manager/carthage/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface ParsedLine {
+  url?: string;
+  type?: string;
```

Comment:
```suggestion
  url: string;
  type: string;
```
Shouldn't `url` and `type` always present?